torch/distributed/tensor/parallel/fsdp.py

```diff
@@ -186,7 +186,7 @@ def _chunk_tensor(
             inner_param,
             rank,
             world_size,
-            torch.cuda.device_count(),
+            torch.get_device_module().device_count(),
```

Comment:
```suggestion
            torch.accelerator.device_count(),
```
On latest trunk you now have this namespace that you can use directly!
Either works here